routemaster/server/endpoints.py

```diff
@@ -58,13 +59,43 @@ def get_state_machines():
         'state-machines': [
             {
                 'name': x.name,
+                'view': f'/state-machines/{x.name}/view',
```

Comment:
I realise you didn't add this originally, but this should probably use `url_for`.

Reply:
You're right, it should. However, since this file predominantly doesn't do that I'm going to leave this for now. I'll open a separate PR to fix that once this merges.

Reply:
https://github.com/thread/routemaster/issues/117